packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -119,6 +121,35 @@ export function createTimeSeries(
   };
 }
 
+function transformResource(
+  resource: Resource,
+  projectId: string
+): { type: string; labels: { [key: string]: string } } {
+  const cloudProvider = `${resource.labels['cloud.provider']}`;
+  // These are the only supported resources
+  if (cloudProvider === 'gcp') {
```

Comment:
Do you think we should define constants for cloudProvider in OT JS and use it here?

Something like:
```js
export const CLOUD_PROVIDER = {
  GCP: 'gcp',
  AWS: 'aws',
  ....
};
```

Reply:
Yeah I can make a followup task with @opentelemetry/resources to add that. Those are the only two cloud providers defined in @opentelemetry/resources.